activity/wswritedata/descriptor.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "wswritedata",
+  "type": "flogo:activity",
+  "version": "0.0.1",
+  "title": "Websocket Write Data Activity",
+  "description": "Websocket Write Data Activity",
+  "homepage": "https://github.com/project-flogo/websocket/tree/master/activity/wswritedata",
+  "settings": [
+    {
+      "name": "uri",
+      "type": "string",
+      "description": "Backend websocket uri to connect",
+      "display": {
```

Comment:
This is not required base on our discussion.  It only using the existing connection and write a message to the connection.

Reply:
removed the redundant code. Done